fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -508,7 +544,7 @@ struct BlobCipherEncryptHeaderRef {
 
 #pragma pack(push, 1) // exact fit - no padding
 typedef struct BlobCipherEncryptHeader {
-	static constexpr int headerSize = 104;
+	static constexpr int headerSize = 112;
```

Comment:
Line 621 has following assert covering the same since the code inception:
```
// Ensure no struct-packing issues
static_assert(sizeof(BlobCipherEncryptHeader) == BlobCipherEncryptHeader::headerSize,
              "BlobCipherEncryptHeader size mismatch");
```

Further, the header is an old format and is replaced with `Configurable Encryption` semantics, the plan is to remove the code sometime in near futures.

Reply:
Ah I missed it, thanks!